pyrate/core/shared.py

```diff
@@ -68,6 +68,7 @@
 class InputTypes(Enum):
     IFG = 'ifg'
     COH = 'coh'
+    BAS = 'bas'
```

Comment:
I understand why you choose "bas" (to conform to the other three character abbreviations like "ifg", "coh", "dem"), but I think "base" might be a bit more readable in the code, and more instantly understandable. There is no real reason to keep these 3 characters, so 4 characters is also fine